modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/storage/SnapshotLocalFileSystemStorage.scala

```diff
@@ -31,6 +33,10 @@ abstract class SnapshotLocalFileSystemStorage[
   path: Path
 ) extends SerializableLocalFileSystemStorage[F, Signed[S]](path) {
 
+  val ordinalChunkSize = ChunkSize(20000)
```

Comment:
Can we move this number to be in configuration?

Reply:
I thought about that and I can do it, but I decided not to. This way, it will be compiled into the code and it won't be possible to change it from configuration (per node) which would help managing the same structure of files across nodes (and for nodectl/archiver too)

Reply:
Hmm, got it, no problem then, I was just curious

Reply:
You can still have it in the configuration classes but have it hardcoded there 🤔  I think we used that pattern before.

Reply:
@kpudlik fair point, I'll take a look